tests/js/roots/partial/index.rst

```diff
@@ -0,0 +1,4 @@
+sphinx_utils module
+===================
+
+Useful utilities.
```

Comment:
Same, I would personally use this opportunity to add a bit of context about what this file tests.